IntelliTect.TestTools.Selenate/IntelliTect.TestTools.Selenate/Browser.cs

```diff
@@ -189,6 +189,20 @@ public void FrameSwitchAttempt(int secondsToWait = 15, params By[] bys)
             }
         }
 
+        /// <summary>
+        /// Disposes the current Selenium Driver
+        /// </summary>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Has Selenium already been disposed?
+        /// </summary>
+        protected bool Disposed = false;
```

Comment:
If you are going to track your disposed state this should be a private field.
```suggestion
        private bool _Disposed;
```

Reply:
Understood, thank you!